core-bundle/src/Resources/contao/drivers/DC_Folder.php

```diff
@@ -714,6 +715,11 @@ public function cut($source=null)
 		}
 		else
 		{
+			$this->denyAccessUnlessGranted(
```

Comment:
this needs to come before the `file_exists`, a user should get access denied and not an overwrite-warning without access rights.

Reply:
Done.